gem/tests/test_auth.py

```diff
@@ -101,6 +102,7 @@ def test_add_superuser_role_from_claims(self):
                           'e2556752-16d0-445a-8850-f190e860dea4')
         self.assertEquals(user.profile.gender, 'f')
         self.assertEquals(user.profile.date_of_birth, date(1988, 5, 22))
+        self.assertEquals(user.profile.alias, 'testgivenname')
```

Comment:
Please also add a check to confirm the alias is set even if it was previously None.